### DISCUSSION TITLE
Remove buttons

### DISCUSSION THREAD

**Discussion by rucksman** [pglombardo/PasswordPusher#1912] [Q&A]
Is there a way to remove the following buttons? I want to close off my pwpush instance (Docker) as much as I can. I did set all the possible settings in settings.yml already, but I want to get rid of these buttons too:
- Forgot your password?
- Didn't receive confirmation instructions?
- Didn't receive unlock instructions?

How can I achieve that? A dirty workaround would be to delete the email settings in settings.yml, but this would be just a workaround.

**Comment by pglombardo** [comment]
Hi @rucksman - With logins enabled, there is no way to remove those buttons but you can limit who can sign up with `PWP__SIGNUP_EMAIL_REGEXP`.  This is documented at the [bottom of this section](https://github.com/pglombardo/PasswordPusher/blob/master/Configuration.md#enabling-logins).

The other potentially helpful option would be `PWP__ALLOW_ANONYMOUS` too.

Beyond that, there are no other options currently.

If you do limit the allowable emails that can be registered, the buttons you listed wouldn't do anything for non-existent accounts.

Does this help?
